mlir/utils/performance/common/benchmarkUtils.h

```diff
@@ -12,6 +12,7 @@
 #define MLIR_UTILS_PERFORMANCE_COMMON_BENCHMARKUTILS_H
 
 #include "hip/hip_runtime.h"
+#include <string>
```

Comment:
what caused this?

Reply:
I was getting build errors about missing templates on my machine without this, figured I might as well stick it in